buildSrc/src/main/kotlin/JpackageExec.kt

```diff
@@ -234,7 +234,7 @@ open class JpackageExec : DefaultTask() {
             fileAssociations.ifPresent { propsFile ->
                 args.addAll("--file-associations", propsFile.asFile.absolutePath)
             }
-            args.addAll("--package-type", installerType.get())
+            args.addAll("--type", installerType.get())
```

Comment:
Why is this change necessary?

Reply:
Because they keep changing flags in jpackage. That's why we're going to a fixed version we're hosting.